### PR TITLE
DEV-519 Integerate drift detection with GitHub App

### PR DESCRIPTION
**Pull request by Resonance1584**

### CODE REVIEW

**Comment by yhakbar** [comment]
@Resonance1584 the action has moved to gruntwork-io

**Review by yhakbar — APPROVED** [review]

File: .github/workflows/pipelines-drift-detection.yml [hunk 1]
```diff
@@ -71,52 +92,95 @@ jobs:
...
-          PIPELINES_READ_TOKEN: ${{ secrets.PIPELINES_READ_TOKEN }}
           JOB_NAME: ${{ env.JOB_NAME }}
           STEP_NAME: "Execute Drift Detection"
 
   consolidate_jobs:
     runs-on: ${{ fromJSON(inputs.runner) }}
     needs: pipelines_drift_detection
     steps:
+      - name: Fetch Gruntwork Read Token
+        id: pipelines-gruntwork-read-token
```
Comment (oredavids) [inline .github/workflows/pipelines-drift-detection.yml:142]: Suggestion: consider removing "token" and or shortening the ids of these fetch steps( e.g fetch_read)

**Review by oredavids — APPROVED** [review]

File: .github/workflows/pipelines-drift-detection.yml (hunk 1 above)
Comment (oredavids) [reply .github/workflows/pipelines-drift-detection.yml:142]: It makes reading the later references easier on the eye

Comment (yhakbar) [reply .github/workflows/pipelines-drift-detection.yml:142]: Personally, I prefer it with the `token` suffix.

Comment (Resonance1584) [reply .github/workflows/pipelines-drift-detection.yml:142]: I don't have a strong preference here, but currently these are consistent with the other workflow files. If this is a strong opinion we should update all of them at once in the v3 branch. Leaving as is for now for consistency as it gets merged.

**Pull request merged**